apis/python/tests/test_sparse_nd_array.py

```diff
@@ -323,6 +260,50 @@ def test_sparse_nd_array_read_as_pandas(
         assert not A.schema.allows_duplicates
 
 
+@pytest.mark.parametrize("appendable", [True, False])
```

Comment:
appendable is not the correct concept, and is very misleading.

We have simply added a default "shape" to the create() method.  All sparse arrays are "appendable" as you are using the term, and _no_ sparse arrays allow their domain/shape to be changed after creation.

For clarity, I suggest picking a different term.  All the code below is doing is incremental writes of different parts of the sparse array (i.e., adding explicit values to the coordanates).

Reply:
Fixed